src/main/java/de/retest/recheck/ui/descriptors/AttributeUtil.java

```diff
@@ -2,8 +2,34 @@
 
 import java.awt.Rectangle;
 
+import de.retest.recheck.ui.diff.AttributeDifference;
+import de.retest.recheck.ui.diff.ElementDifference;
+import de.retest.recheck.ui.diff.IdentifyingAttributesDifference;
+
 public class AttributeUtil {
 
+	public static Rectangle getActualOutline( final ElementDifference difference ) {
+		final IdentifyingAttributesDifference diff =
+				(IdentifyingAttributesDifference) difference.getIdentifyingAttributesDifference();
+		for ( final AttributeDifference aDiff : diff.getAttributeDifferences() ) {
+			if ( aDiff.getKey().equals( OutlineAttribute.RELATIVE_OUTLINE ) ) {
+				return ((Rectangle) aDiff.getActual());
+			}
+		}
+		return getOutline( difference.getIdentifyingAttributes() );
+	}
+
+	public static Rectangle getActualAbsoluteOutline( final ElementDifference difference ) {
```

Comment:
`getActualOutline` and this method are quite similar. Does it make sense to merge/reuse parts?